### DISCUSSION TITLE
please help me. what's meaning this error message.

### DISCUSSION THREAD

**Discussion by fanuser** [rabbitmq/rabbitmq-server#6296] [Other]
please tell me about this error message.
{bad_header,<<1,0,0,0,0,0,11,0>>}
and How to Troubleshoot Errors

**Comment by michaelklishin** [comment]
A client of some kind sent the byte sequence `1,0,0,0,0,0,11,0` to a port that expected a different protocol handshake.

Most likely you have pointed an HTTP API client at port 5672 (5671 with TLS) or an AMQP 0-9-1 (AMQP 1.0, etc) at port 15672. It's impossible to tell from just one line. Consider always providing as much context as possible when seeking help.

**Reply by fanuser** [reply]
thinks 
I have solved the problem

**Comment by Roikai** [comment]
您好，这个问题是什么原因呢，我英语不好，不太理解外国友人的回答。   您是如何解决这个问题的。  thanks

**Reply by fanuser** [reply]
他说是请求的协议错了，你可以tcp抓包一下看看到底给rabbitmq发的是什么信息。

**Comment by Roikai** [comment]
好的。非常感谢您的回复

蔡政 ***@***.***> 于2022年12月14日周三 19:17写道：
